### DISCUSSION TITLE
While writing local site rebuilds with every few keystrokes, any way to configure a little more harmony?

### DISCUSSION THREAD

**Discussion by gingerchew** [0xsuk/agitcms#33] [Q&A] [answered]
First off, this is incredible.

Second, I'm using 11ty as my SSG and because Agit is updating the file every few keystrokes, it is causing the server/site to rebuild very frequently.

It isn't a huge deal since my site (ginger.wtf) is less than 20 pages, but figured it was worth asking anyways. I love that because it is manipulating the file directly, it is basically auto-saving, but I wonder if having an option for a save button would be possible.

Edit: Turning on 11ty's incremental feature prevents the *entire site* rebuilding, but question still stands.

Seriously loving this CMS, totally fit my needs and was super simple set up! Cheers!

**Comment by 0xsuk** [comment, marked as answer]
Thanks for the idea. Yes, Agit saves every single keystroke to a file, which I also think is not a good idea. Saving configuration will be tweakable at the next release.
